solutions/ac-GCDOnBlackboard.mdx

```diff
@@ -76,4 +76,45 @@ int main(){
 
 ```
 </CPPSection>
+	
+<JavaSection>
+
+```java
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
+        int N = Integer.parseInt(input.readLine());
+        int[] nums = new int[N];
+        String[] line2 = input.readLine().split(" ");
```

Comment:
comments would be nice

Reply:
can't you just use string tokenizer instead?

Reply:
also a better name than just `line2`?